atlp-rwanda/elites-bn-be
# fixing backend to satisfy frontend

# What does this PR do?
- This is PR for creating login and register to generate role Id in token 
# Description of Task to be completed?
- add user roleId in payload
# How should this be manually tested?
- register a user , copy token and use jwt.io to decrypt token , after check the plain text if it has role

# Screenshots (if appropriate)
<img width="1188" alt="Screen Shot 2022-05-06 at 12 54 08" src="https://user-images.githubusercontent.com/79087014/167118656-215ad2b0-55c5-4d3c-9779-94b5598cb995.png">
# Questions:
